functions/src/testimony/publishTestimony.ts

```diff
@@ -58,7 +60,23 @@ class PublishTestimonyTransaction {
       court: this.draft.court,
       position: this.draft.position,
       version: await this.getNextPublicationVersion(),
-      publishedAt: Timestamp.now()
+      publishedAt: Timestamp.now(),
+      // representativeId: this.profile.representative.Id,
+      // senatorId: this.profile.senator.Id,
+      // senatorDistrict: this.profile.senator.district,
+      // representativeDistict: this.profile.representative.district,
+    }
+    if (this.profile.representative.id) {
```

Comment:
Hey @jkoren , I think the integration tests are failing because `profile` is undefined in these conditionals, since the test user doesn't have legislators selected. We can address this by using the `?.` operator: `if (this.profile?.representative.id)` etc. for these 4 conditionals.

Reply:
That makes sense.  I'll added the ?. after senator and representative in case the user chooses only one representative and not both (this.profile?.representative?.id)